docs/frontend-api/introduction-to-frontend-api.md

```diff
@@ -306,3 +306,27 @@ YourInput:
 !!! note
 
     Check the corresponding decorator file in `packages/frontend-api/src/Resources/config/graphql-types/` to see which constraints you need to include.
+
+#### Removing or Replacing Validation Constraints
+
+If you want to remove or replace a validation constraint from the decorator, you can disable it by assigning a non-existent validation group.
+This effectively skips the constraint because the group will never be active during validation.
+
+```yaml
+# project-base/app/config/graphql/types/.../YourInput.types.yaml
```

Comment:
The project-base folder should not be mentioned. The context for the users starts right in the project-base folder